bftengine/src/bcstatetransfer/RVBManager.cpp

```diff
@@ -225,10 +230,12 @@ void RVBManager::updateRvbDataDuringCheckpoint(CheckpointDesc& new_checkpoint_de
                                              last_checkpoint_desc_.maxBlockId));
   ConcordAssertAND((last_checkpoint_desc_.maxBlockId <= new_checkpoint_desc.maxBlockId),
                    (last_checkpoint_desc_.checkpointNum < new_checkpoint_desc.checkpointNum));
+  metrics_.checkpoint_num_.Get().Set(new_checkpoint_desc.checkpointNum);
 
   //
   //    Add blocks to RVT
   //
+  // TBD MD duration tracker time spent to add blocks to RVT
```

Comment:
Is this a TODO?
if yes, better mark it as 
`// TODO - ... `

Also, I don't think we want a duration tracker for this, it should be a historgram.
As of now the RVT is really small and we are dealing here in times of microsecs while we have operations that takes tens and hunderds of millisecs. So this is not an issue for now.. I'm not saying it's bad to add it, but it's not going to be a performance issue, its a very wide tree and we have a quite effiient insertion/deletion mechanism. we dont even need to traverse the tree.

Reply:
Thanks. I've removed it.